lib/modules/manager/nuget/update.spec.ts

```diff
@@ -0,0 +1,79 @@
+// TODO #7154
```

Comment:
@Edouard-chin can you disable those messages by default?

@rarkins Should we disable / remove the catana app?

Reply:
Sorry for the unwanted noise. I think it will make sense you uninstall Catana from renovate as you aren't using it. Let me know if you want me to uninstall it for you!